docs/en/observability/ci-cd-observability.asciidoc

```diff
@@ -427,3 +427,58 @@ The Concourse CI doesn’t report health metrics through OpenTelemetry. However,
 https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/spanmetricsprocessor#span-metrics-processor[OpenTelemetry Collector Span Metrics Processor]
 to derive pipeline execution traces into KPI metrics like throughput and the error rate
 of pipelines.
+
+
+[discrete]
+[[check-service-health-from-cd-pipelines]]
+== Check service health from deployment pipelines
+
+
+Integrating automated service health checks in deployment pipelines is critical for end-to-end deployment automation, which crucially enables deployment frequency to be increased.
+
+Elastic Observability exposes service HTTP health check APIs that can be integrated in deployment pipelines to verify the behaviour of newly deployed instances, and either automatically continue the deployments or rollback according to the health status.
```

Comment:
@ollyhowell should we sazy "service HTTP health check APIs" or "HTTP APIs to check the health of services"?

Reply:
I think the original version is better unless you want to break the sentence down into a couple of sentences. 

With the proposed change, you end up with  "...HTTP APIs to check the health of services that can be integrated..." (the modifier ends up applying to the wrong word).